lib/meta-box-partial-page.php

```diff
@@ -144,39 +147,66 @@ function gutenberg_meta_box_partial_page( $post_type, $meta_box_context ) {
  * @param WP_Locale $wp_locale      Locale object.
  */
 function gutenberg_meta_box_partial_page_admin_header( $hook_suffix, $current_screen, $wp_locale ) {
-	/* Scripts and styles that meta boxes can potentially be using */
-	wp_enqueue_style( 'common' );
-	wp_enqueue_style( 'buttons' );
-	wp_enqueue_style( 'colors' );
-	wp_enqueue_style( 'ie' );
-
+	/* Scripts that meta boxes can potentially be using */
 	wp_enqueue_script( 'utils' );
 	wp_enqueue_script( 'common' );
 	wp_enqueue_script( 'svg-painter' );
+	?>
+	<!-- Add an html class so that scroll bars can be removed in css and make it appear as though the iframe is one with Gutenberg. -->
+	<html class="gutenberg-meta-box-html">
```

Comment:
No styles exist anymore for this class name, we can drop it?

Reply:
Yes